src/cython/vapoursynth.pyx

```diff
@@ -1225,6 +1225,36 @@ cdef FrameProps createFrameProps(RawFrame f):
 Mapping.register(FrameProps)
 
 
+class ChannelLayout(int):
+    def __contains__(self, layout):
+        return bool(self & (1 << layout))
+
+    def __iter__(self):
```

Comment:
Replace this function with an appropriate `__getattr__`, and python will automatically write the following `__iter__` implementation for you:

```py
def __iter__(self):
    for i in range(len(self)):
        yield self[i]
```